python/lsst/analysis/tools/actions/plot/focalPlanePlot.py

```diff
@@ -504,10 +557,26 @@ def makePlot(
         ax.tick_params(labelsize=7)
 
         ax.set_aspect("equal")
+
+        if self.addHistogram:
```

Comment:
I think this function is long enough to justify splitting into single-underscore methods. For example, the 45 lines from `divider = make_axes_locatable(ax)` to `fig = addPlotInfo(fig, plotInfo)` are identical except for the arg passed to `fig.colorbar`.